translations/translation_pt.xml

```diff
@@ -8,418 +8,417 @@
-    <text name="CP_global_setting_drawOntoTheHudMap_title" text="Schows courses on the minimap"/>
-    <text name="CP_global_setting_drawOntoTheHudMap_tooltip" text="Enables drawing of the course onto the mini map."/>
-    <text name="CP_global_setting_infoTextHudActive_title" text="Mostrar janela de textos informativos"/>
-    <text name="CP_global_setting_infoTextHudActive_tooltip" text="Exibe uma janela de texto informativo."/>
-    <text name="CP_global_setting_infoTextHudPlayerMouseActive_title" text="Evento do mouse do jogador"/>
-    <text name="CP_global_setting_infoTextHudPlayerMouseActive_tooltip" text="Ativa/desativa o evento do mouse do jogador para a exibição do texto informativo fora de um veículo."/>
-    <text name="CP_global_setting_preferCustomFields_title" text="Prefira campos personalizados"/>
-    <text name="CP_global_setting_preferCustomFields_tooltip" text="Prefira o campo personalizado se também houver um campo regular na posição selecionada."/>
-    <text name="CP_global_setting_distanceUnit_title" text="Distance unit"/>
-    <text name="CP_global_setting_distanceUnit_tooltip" text="Distance unit meter or Foot."/>
-    <text name="CP_global_setting_distanceUnit_meter" text="Meter"/>
-    <text name="CP_global_setting_distanceUnit_imperial" text="Foot"/>
+    <text name="CP_global_setting_fuelThreshold_tooltip" text="O motorista irá parar, se a percentagem de combustível for menor."/>
+    <text name="CP_global_setting_brokenThreshold_title" text="Limite de danos"/>
+    <text name="CP_global_setting_brokenThreshold_tooltip" text="O trabalhador irá parar, se o veículo ou a alfaia tiverem mais dano. Os 100% indica que não aciona este opção."/>
+    <text name="CP_global_setting_subTitle_userSettings" text="Configurações do Utilizador"/>
+    <text name="CP_global_setting_controllerHudSelected_title" text="Ecrã secundário de opções acionado pelo controlador de jogo"/>
+    <text name="CP_global_setting_controllerHudSelected_tooltip" text="Usa e navega no ecrã secundário de opções com o controlador de jogo."/>
+    <text name="CP_global_setting_showActionEventHelp_title" text="Alerta de eventos"/>
+    <text name="CP_global_setting_showActionEventHelp_tooltip" text="Mostra o texto de ajuda com um alerta."/>
+    <text name="CP_global_setting_subTitle_pathfinder" text="Configurações do encontrar caminhos"/>
+    <text name="CP_global_setting_maxDeltaAngleAtGoal_title" text="Ângulo diferente para o objectivo"/>
+    <text name="CP_global_setting_maxDeltaAngleAtGoal_tooltip" text="Diferença máxima autorizada do ângulo até ao objectivo, valores mais altos indica que vai mais rápido, mas com menos exacto"/>
+    <text name="CP_global_setting_deltaAngleRelaxFactor_title" text="Factor de tolerância"/>
+    <text name="CP_global_setting_deltaAngleRelaxFactor_tooltip" text="Factor para diminuir a precisão de encontrar o caminho com iterações mal sucedidas, valores maiores são mais rápidos"/>
```

Comment:
Please revert those changes:
CP_global_setting_maxDeltaAngleAtGoal_title
CP_global_setting_maxDeltaAngleAtGoal_tooltip
CP_global_setting_deltaAngleRelaxFactor_title
CP_global_setting_deltaAngleRelaxFactor_tooltip

You did remove the Deg, wich leads into missing translation.